### ISSUE TITLE
i cant find "You will need to enable "Support adding custom themes" in the Labs section of Settings to make these themes appear in the Appearance section of Settings."

### ISSUE THREAD

**Issue by DARKDRAGON532** [aaronraimist/element-themes#32]
has it been removed?

**Comment by aaronraimist** [comment]
No it is still there. Make sure you are using the current version of Element (1.7.30) and you are looking at the Labs settings.

**Comment by DARKDRAGON532** [comment]
well i fixed it, im an idiot so sorry for that...

**Comment by Roboron3042** [comment]
I'm using Element 1.7.34 and I can't see this setting in the Lab settings (I only see the Spaces beta). Has something changed?

**Comment by aaronraimist** [comment]
@Roboron3042 to start with you should update to the latest version, if only because there were security fixes in 1.8.4.

Second, read through https://github.com/aaronraimist/element-themes#how-to-use-themes. If you only see Spaces then you haven't actually enabled labs. Yes it is very confusing, they made spaces show up even for people who didn't enable labs. Usually the labs tab would only show up when it is enabled.

**Comment by Roboron3042** [comment]
Forget it. I did add those changes to the config before but the problem is that Element was still using Riot config path...
```
/home/rober/.config/Element exists: no
/home/rober/.config/Riot exists: yes
Using legacy user data path: /home/rober/.config/Riot
```
(even though ~/.config/Element did exist...)

I have now enabled them with no problem.
